### ISSUE TITLE
Columbus GovDelivery feed is OHCC_64

### ISSUE THREAD

**Issue by ohiofi** [benlk/columbus-govdelivery-rss#2]
I don't know where I found it, but I stumbled upon the RSS feed for Columbus GovDelivery

https://public.govdelivery.com/topics/OHCCC_64/feed.rss

Hope that helps?

**Comment by benlk** [comment]
That appears to be the Rec & Parks weekly updates feed: https://github.com/benlk/columbus-govdelivery-rss/blob/5505e3b64d1ff41342ebd2e04e22d6266d828098/feeds.csv#L328

**Comment by ohiofi** [comment]
My mistake. I understand what happened now...

I was in the process of moving a Mastodon bot from botsin.space to a different instance. The bot was using a govdelivery rss feed for OHCCC, but I couldn't remember the exact feed URL. I googled something like "columbus govdelivery rss" and found your GitHub repo. I saw your "Oddities" section. A few minutes later, I found the rss feed that my bot was using and I mistakenly assumed that I had *solved* the mystery of the main OHCCC mailing list.
